WalletWasabi.Gui/Controls/WalletExplorer/CoinListViewModel.cs

```diff
@@ -72,6 +71,8 @@ public class CoinListViewModel : ViewModelBase
 
 		public ReadOnlyObservableCollection<CoinViewModel> Coins => _coinViewModels;
 
+		public bool DisplayWarningAboutCommonOwnership { get; set; } = false;
```

Comment:
How about to rename this to `DisplayCommonOwnershipWarning`

Reply:
Yes, that's a better name.